docs/DafnyRef/Expressions.md

```diff
@@ -518,7 +518,26 @@ type `nat -> T` where `T` is the array element type:
 ```dafny
 var a := new int[5](i => i*i);
 ```
-TODO: what about multi-dimensional arrays
+
+To allocate a multi-dimensional array, simply give the sizes of
```

Comment:
How did you pick a line-width here? Is there also a case to be made for using soft-wrapping?

Reply:
I didn't give the matter any thought. I guess I just formatted the text with a line-width similar to that of the surrounding paragraphs. It makes no difference for the rendered markdown. I like these broken-up lines in the `.md` file, but I'm open to a refactoring of the entire reference manual to use soft-wrapping instead.